tests/run-make/asm-incr-cache/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ only-x86_64-unknown-linux-gnu
```

Comment:
Why is this limited to x86_64 linux? Nothing in this test looks to be platform dependent.

Reply:
Good question, probably from the test I adopted. Removed, let's see.